src/Sarif/SarifUtilities.cs

```diff
@@ -20,14 +20,12 @@ public static bool IsSemanticVersioningCompatible(this string versionText)
             return s_semVer200.IsMatch(versionText);
         }
 
-        private const string V1_0_0 = "1.0.0";
-        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
-        private const string V2_0_0 = "2.0.0";
+        public const string V1_0_0 = "1.0.0";
+        public const string V1_0_0_BETA_5 = "1.0.0-beta.5";
```

Comment:
>V1_0_0_BETA_5 [](start = 28, length = 13)

Nobody uses this. We should remove it.

Reply:
go right ahead

---
In reply to: [221469436](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221469436) [](ancestors = 221469436)

Reply:
Fixed.

---
In reply to: [221750460](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221750460) [](ancestors = 221750460,221469436)